cft/deploy.yml

```diff
@@ -0,0 +1,38 @@
+Resources:
+  ElasticsearchDomain:
+    Type: AWS::Elasticsearch::Domain
+    Properties:
```

Comment:
Coverity found issue: Missing TLS - CWE-319, Low Severity

****

**How to fix:** Enable node-to-node encryption by explicitly setting the `Properties.NodeToNodeEncryptionOptions.Enabled` value to `true`.

<details>
<summary>Click to expand data flow...</summary>

**From cft/deploy.yml:**
```
    1 Resources:
    2   ElasticsearchDomain:
    3     Type: AWS::Elasticsearch::Domain
1. Sigma main event: The Amazon Elasticsearch Service (Amazon ES) domain does not use TLS for node-to-node encryption within the VPC.

2. remediation: Enable node-to-node encryption by explicitly setting the `Properties.NodeToNodeEncryptionOptions.Enabled` value to `true`.

    4     Properties:
    5       DomainName: "test"
    6       ElasticsearchClusterConfig:
```
</details><!-- Coverity 787b9ad277c81a9b66c83f23bf1bf6c4 -->